vips/vip-262/bscmainnet.ts

```diff
@@ -0,0 +1,115 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+export const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+export const USDT = "0x55d398326f99059fF775485246999027B3197955";
+export const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+export const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+export const vUSDC = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+export const vUSDT = "0xfD5840Cd36d94D7229439859C0112a4185BC0255";
+export const vETH = "0xf508fCD89b8bd15579dc79A6827cB4686A3592c8";
+export const USDC_AMOUNT = parseUnits("7587064.24", 18);
+export const USDT_AMOUNT = parseUnits("377487.92", 18);
+export const ETH_AMOUNT = parseUnits("126.6523", 18);
+
+export const vip262 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-262 Treasury Management",
+    description: `#### Summary
+
+Following the community proposal for [Treasury Managment](https://community.venus.io/t/treasury-management-proposal/4134), and the [associated snapshot](https://snapshot.org/#/venus-xvs.eth/proposal/0xb58cdfa3dba5459e9279a06065033c327f239afb834b27e06c5474954e51b30d), this VIP, If approved, will supply 7,587,064.24 USDC, 377,487.92 USDT and 126.6523 ETH from the vTreausry to the Venus market.
+
+These actions are projected to generate approximately $1,057,435 annual revenues considering market prices and conditions for Feb 21, 2024.
```

Comment:
The amounts are different from the ones in the snapshot vote but the projected revenue is the same. Is it intended?